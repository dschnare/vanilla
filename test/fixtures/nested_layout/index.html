<extends "./page.html">

<replace body>
  <p>This is the body!</p>
</replace>

<append body>
  <p>This is more content added to the body.</p>
</append>

<block any>
  this should be ignored.
</block>