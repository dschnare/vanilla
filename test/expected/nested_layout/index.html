<html>
  <head>
    <title>Index</title><meta charset="utf-8" />
  </head>
  <body>
    
    <p>This is the body!</p><p>This is more content added to the body.</p>
    
  </body>
</html>